README.md

```diff
@@ -232,17 +232,17 @@ If you are working with Azure Container Services or Azure Kubernetes Services, t
 
 If you move your configuration file between machines with different OSes (and therefore different paths to binaries) you can override the following settings on a per-OS basis by appending `.windows`, `.mac` or `.linux` to the setting name:
 
-  * `vs-kubernetes.kubectl-path`
-  * `vs-kubernetes.helm-path`
-  * `vs-kubernetes.minikube-path`
+  * `vscode-kubernetes.kubectl-path`
+  * `vscode-kubernetes.helm-path`
+  * `vscode-kubernetes.minikube-path`
```

Comment:
They should not be changed as we are talking about the `vs-kubernetes` section here. This is the old way to set 
 paths.

Reply:
Yes, but if you want to update the readme you need to describe how it works correctly.
In the beginning it mentions that to configure the setting you have to use ` vs-kubernetes -> vs-kubernetes.${tool}-path configuration setting.`. Then you mention `vscode-kubernetes.kubectl-path`, and finally the section `vscode-kubernetes`.
So either you remove the readme change and it will be covered in another PR or you update the entire section.